boards/DCM/Inc/Io/Io_LSM6DS33.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+/**
+ * Driver for interfacing with the IMU LSM6DS33 via I2c
+ *
+ * The Application Note for this IMU can be found here:
+ * https://www.pololu.com/file/0J1088/LSM6DS33-AN4682.pdf
+ */
+
+#include <stdint.h>
+#include <stm32f3xx_hal.h>
+#include <stdbool.h>
+
+struct ImuData;
+
+/* Function Definitions */
+/**
+ * Configures the LSM6DS33 IMU Gyro/Accelerometer chip
+ *
+ * @param      i2c_handle The handle for the I2C device the IMU is connected to
+ * @return     None
```

Comment:
The formatting looks weird here. I think running Rebuild 'DCM.elf' should take care of this. If not make sure that there's only a single space between @param and its description

Reply:
It didn't help but I changed it manually